dockerfiles/site/website/compile.sh

```diff
@@ -39,6 +39,8 @@ git checkout $SHA1
 
 # Compilation
 
+export MAVEN_OPTS="-Xmx7168m -Xms2048m -XX:MaxPermSize=256m -XX:+UseConcMarkSweepGC -XX:-UseGCOverheadLimit"
```

Comment:
that's huge. Could you try with maven 3.5 ? (I read that they fix a memory problem when nb modules is big)